### PR TITLE
 fix(function): Update MODS to teamsfx

### PR DESCRIPTION
**Pull request by henzhang-ms**

### CODE REVIEW

File: function-base/js/default/README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
-# Build Teams Application Backend with MODS and Azure Functions
+# Build Teams Application Backend with Teamsfx and Azure Functions
```
Comment (IvanJobs) [inline function-base/js/default/README.md:1]: Nit: may need to confirm Teamsfx or TeamsFx, or even something else?

File: function-base/js/default/README.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+When building a Teams application, teamsfx provides an option for you to add a backend API to develop server-side logics so that you can easily build your systems to react to a series of critical events. The API you added is actually an [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/) project that handles HTTP requests from Tabs, and you can customize it according to your requirements.
 
 ## Prerequisites
 
 To start enjoying full functionalities to develop an API with Azure Functions for your Teams Application, you need to:
 - Install [Azure Functions Core Tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Ccsharp%2Cbash).
-- Install [MODS Server SDK Package](https://aka.ms/MODSPrivatePreview/server-sdk).
-- Add an API during project creation or using command, see [MODS User Manual](https://mods-landingpage-web.azurewebsites.net/md/guide/index).
+- Install [Teamsfx Server SDK Package](https://aka.ms/MODSPrivatePreview/server-sdk).
+- Add an API during project creation or using command, see [Teamsfx User Manual](https://mods-landingpage-web.azurewebsites.net/md/guide/index).
```
Comment (IvanJobs) [inline function-base/js/default/README.md:10]: mods-landingpage-web?

File: function-base/js/default/README.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
+By default, teamsfx will provide template code for you to get started. The starter code handles calls from your Teams App client side, initializes the teamsfx server SDK to access current connected user information and prepares a pre-authenticated Microsoft Graph Client for you to access more user's data. You can modify the template code with your custom logics or add more functions with `HTTPTrigger` by running command `MODSV2 - Add Resource` and select `New APIs from Azure Functions`. Read on [Azure Functions developer guide](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference) for more development resources.
```
Comment (IvanJobs) [inline function-base/js/default/README.md:14]: MODSV2 - Add Resource?

File: function-base/js/default/README.md (hunk 1 above)
Comment (henzhang-ms) [reply function-base/js/default/README.md:1]: The update in SDK is teamsfx, this is title so upper case the first letter.

File: function-base/js/default/README.md (hunk 2 above)
Comment (henzhang-ms) [reply function-base/js/default/README.md:10]: yeah, I think the link should be updated after new link is ready.

File: function-base/ts/default/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+When building a Teams application, teamsfx provides an option for you to add a backend API to develop server-side logics so that you can easily build your systems to react to a series of critical events. The API you added is actually an [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/) project that handles HTTP requests from Tabs, and you can customize it according to your requirements.
+
+## Prerequisites
+
+To start enjoying full functionalities to develop an API with Azure Functions for your Teams Application, you need to:
+- Install [Azure Functions Core Tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Ccsharp%2Cbash).
+- Install [Teamsfx Server SDK Package](https://aka.ms/MODSPrivatePreview/server-sdk).
+- Add an API during project creation or using command, see [Teamsfx User Manual](https://mods-landingpage-web.azurewebsites.net/md/guide/index).
+
```
Comment (IvanJobs) [inline function-base/ts/default/README.md:11]: Nit: It seems like dependent links.

**Review by IvanJobs — APPROVED** [review]

File: function-base/js/default/README.md (hunk 1 above)
Comment (henzhang-ms) [reply function-base/js/default/README.md]: Updated to TeamsFX

File: function-base/js/default/README.md (hunk 2 above)
Comment (henzhang-ms) [reply function-base/js/default/README.md]: Here updates the label. I will update the link after they are ready

File: function-base/js/default/README.md (hunk 3 above)
Comment (henzhang-ms) [reply function-base/js/default/README.md]: Updated

File: function-base/js/default/README.md [hunk 5]
```diff
@@ -25,7 +25,7 @@ By default, MODS will provide template code for you to get started. The starter
 - Deploy your project to the Azure function app by running command - `MODSV2 - Deploy Package` and select `Azure Function`.
```
[the remark this answers is not in the document]
Comment (henzhang-ms) [reply function-base/js/default/README.md]: Updated

File: function-base/js/default/README.md [hunk 6]
```diff
@@ -25,7 +25,7 @@ By default, MODS will provide template code for you to get started. The starter
...
 
 ## Node version
-The runtime versions supported by Azure Functions are list [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions). By default, MODS toolkit provisions an Azure function app with function runtime version 3, and node runtime version 12. You can change the node version through Azure Portal.
+The runtime versions supported by Azure Functions are list [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions). By default, teamsfx toolkit provisions an Azure function app with function runtime version 3, and node runtime version 12. You can change the node version through Azure Portal.
 
 - Sign in [Azure Portal](https://azure.microsoft.com/).
 - Find the Azure function app. The resource group name and the Azure function app name are recorded in the file '.mods/env.*.json'. You can find them by searching the key 'resourceGroupName' and 'functionAppName' in that file.
```
[the remark this answers is not in the document]
Comment (henzhang-ms) [reply function-base/js/default/README.md:31]: It seems we still use .mods in new release. I will update it once I noticed the change.

File: function-base/ts/default/README.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+# Build Teams Application Backend with Teamsfx and Azure Functions
+
+When building a Teams application, teamsfx provides an option for you to add a backend API to develop server-side logics so that you can easily build your systems to react to a series of critical events. The API you added is actually an [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/) project that handles HTTP requests from Tabs, and you can customize it according to your requirements.
+
+## Prerequisites
+
+To start enjoying full functionalities to develop an API with Azure Functions for your Teams Application, you need to:
+- Install [Azure Functions Core Tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Ccsharp%2Cbash).
+- Install [Teamsfx Server SDK Package](https://aka.ms/MODSPrivatePreview/server-sdk).
```
[the remark this answers is not in the document]
Comment (henzhang-ms) [reply function-base/ts/default/README.md]: ok.

**Pull request merged**